iroh-net/src/endpoint.rs

```diff
@@ -405,7 +405,7 @@ impl Endpoint {
             Arc::new(quinn::TokioRuntime),
         )?;
         trace!("created quinn endpoint");
-
+        debug!(version = env!("CARGO_PKG_VERSION"), "iroh Endpoint created");
```

Comment:
Open Q: Do we still need the trace log?

Reply:
what about commit version?

Reply:
Also I'm not sure if my linked answer would work with libraries. :thinking:

Reply:
> Open Q: Do we still need the trace log?

the trace logs distinguish magicsock creation from quinn endpoint creation.  i don't know right now if that's useful or not.  but it's trace so left it alone.

Reply:
git commit sounds useful, but I don't know how to do it - as shown by @matheus23 's responses.  happy to add it if there's a simple recipe though.